compose/cli/main.py

```diff
@@ -238,15 +238,15 @@ def rm(self, project, options):
         Usage: rm [options] [SERVICE...]
 
         Options:
-            --force   Don't ask to confirm removal
-            -v        Remove volumes associated with containers
+            -f, --force   Don't ask to confirm removal
+            -v            Remove volumes associated with containers
         """
         all_containers = project.containers(service_names=options['SERVICE'], stopped=True)
         stopped_containers = [c for c in all_containers if not c.is_running]
 
         if len(stopped_containers) > 0:
             print("Going to remove", list_containers(stopped_containers))
-            if options.get('--force') \
+            if options.get('--force') or options.get('-f') \
```

Comment:
You are entirely right. I seem to have fixed a bug that was fixed! Sorry, I also tested this against my current version of fig where the flag failed and thought the above had fixed it. I can close this pull request if that makes the most sense.